src/controllers/university-controller.js

```diff
@@ -91,12 +103,14 @@ router.post("/", async (req, res) => {
         const city = req.body.city
 
         if (!name || !city) {
-            return res.status(400).json({ error: "To create a new university, name and city are required." });
+            // Translation: To create a new university the following information are required: name, city
+            return res.status(400).json({ error: "새로운 대학교를 등록하기 위해서 다음 정보가 필요합니다: 이름, 도시" });
         }
         const existingUniversity = await Universities.findOne({ name: name });
 
         if (existingUniversity) {
-            return res.status(409).json({ error: "The following university is already registered in the system." });
+            // Translation: The following university is already registered in the system
+            return res.status(409).json({ error: "다음 대학교는 이미 시스템에 등록되어 있습니다." });
         }
```

Comment:
## Possible Problem
Though this would work if the administrator sends in a university in the exact same name, the same university can be expressed in several names (University of Gothenburg, 예테보리 대학교 etc.). This would be problematic in the future
## Possible Solution
Make a dictionary or something similar to avoid duplicate programs from being registered to our DB.

Reply:
This will be addressed when fixing the problem for Program Controller POST